### PR TITLE
Document Social Science One

### PR DESCRIPTION
**Pull request by BerylBogui**

### CODE REVIEW

**Comment by michelbl** [comment]
Looks good to me.

File: _pages/encyclopedia/6-research/3-tools.md [hunk 1]
```diff
@@ -20,7 +20,15 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
 
 ## Facebook Social Science One
 
-<!-- à rédiger -->
+[Social Science One](https://socialscience.one/) is an organization of academics created by Harvard Professor and Director of the Harvard Institute for Quantitative Social Science Gary King and Stanford Law School Professor Nathaniel Persily.
```
Comment (MattiSG) [inline _pages/encyclopedia/6-research/3-tools.md]: No.
The base for operation is described in a paper by King & Persily, but SS1 is not an organisation of academics. It is “an LLC operating on a not-for-profit basis” that “ implements a new type of partnership between academic researchers and private industry”.

File: _pages/encyclopedia/6-research/3-tools.md [hunk 2]
```diff
@@ -20,7 +20,15 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
+
+It implements **partnerships between academic researchers and private industry actors**, granting the former access to relevant information amassed by the latter, who can "_enlist the scientific community to improve their business and produce social good, while protecting their competitive positions_". 
+
+The structure of the partnerships enables a **commission of trusted academics** to obtain "_relevant company information and systems_" and to **invite other independent scholars** to do "_research in specific areas, following standard peer review protocols, funded by nonprofit foundations, and with no required pre-publication approval by the company_". 
+
+The process aims to "_understand and ameliorate society’s greatest challenges_" through social research based on useful yet too often held back data, all the while **ensuring customer privacy**_".
```
Comment (MattiSG) [inline _pages/encyclopedia/6-research/3-tools.md]: Syntax issue, trailing underscore.

File: _pages/encyclopedia/6-research/3-tools.md [hunk 3]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
+
+[Companies](https://socialscience.one/faq/how-can-my-company-work-social-science-one) collecting informative social data can apply by directly [contacting](https://gking.harvard.edu/pages/contact) the co-founders.
+
+These independent scholars must go through and pass a rigorous peer review process before being allowed to access data and [funding](https://socialscience.one/grant-process). Academics can **apply** for grants by answering formal Requests for Proposals (RFPs) issued by the [Social Science Research Council](https://www.ssrc.org/) and whose criteria are defined by SS1. Funding originates from several charitable foundations but is for now independently administered by the SSRC. The SSRC also leads the peer review process and manages the research along with SS1.
+
+Updates are regularly posted on SS1's [blog](https://socialscience.one/blog).
+
+The process aims to "_understand and ameliorate society’s greatest challenges_" through social research based on useful yet too often held back data, all the while **ensuring customer privacy**_".
```
Comment (Ndpnt) [inline _pages/encyclopedia/6-research/3-tools.md]: missing opening `"_`:  `**ensuring customer privacy**_"`  --> `"_**ensuring customer privacy**_"`

Comment (Ndpnt) [reply _pages/encyclopedia/6-research/3-tools.md]: I'm on it

File: _pages/encyclopedia/6-research/3-tools.md [hunk 4]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:35]:
```suggestion
The process aims to "_understand and ameliorate society’s greatest challenges_" through social research based on useful yet too often held back data, all the while _**ensuring customer privacy**_". 
```

File: _pages/encyclopedia/6-research/3-tools.md [hunk 5]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
 
 ## Facebook Social Science One
 
-<!-- à rédiger -->
+[Social Science One](https://socialscience.one/) is a non-profit limited liability company (LLC) based on the [work](http://garyking.org/partnerships) of academics Gary King (Harvard Professor and Director of the Harvard Institute for Quantitative Social Science) and Nathaniel Persily (Stanford Law School Professor).
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:23]:
```suggestion
[Social Science One](https://socialscience.one/)(SS1) is a non-profit limited liability company (LLC) based on the [work](http://garyking.org/partnerships) of academics Gary King (Harvard Professor and Director of the Harvard Institute for Quantitative Social Science) and Nathaniel Persily (Stanford Law School Professor).
```
Abbreviations must be introduced

File: _pages/encyclopedia/6-research/3-tools.md [hunk 6]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
+
+[Companies](https://socialscience.one/faq/how-can-my-company-work-social-science-one) collecting informative social data can apply by directly [contacting](https://gking.harvard.edu/pages/contact) the co-founders.
+
+These independent scholars must go through and pass a rigorous peer review process before being allowed to access data and [funding](https://socialscience.one/grant-process). Academics can **apply** for grants by answering formal Requests for Proposals (RFPs) issued by the [Social Science Research Council](https://www.ssrc.org/) and whose criteria are defined by SS1. Funding originates from several charitable foundations but is for now independently administered by the SSRC. The SSRC also leads the peer review process and manages the research along with SS1.
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:31]:
```suggestion
These independent scholars must go through and pass a rigorous peer review process before being allowed to access data and [funding](https://socialscience.one/grant-process). Academics can **apply** for grants by answering formal Requests for Proposals (RFPs) issued by the [Social Science Research Council](https://www.ssrc.org/)(SSRC) and whose criteria are defined by SS1. Funding originates from several charitable foundations but is for now independently administered by the SSRC. The SSRC also leads the peer review process and manages the research along with SS1.
```

File: _pages/encyclopedia/6-research/3-tools.md [hunk 7]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
+
+Updates are regularly posted on SS1's [blog](https://socialscience.one/blog).
+
+The process aims to "_understand and ameliorate society’s greatest challenges_" through social research based on useful yet too often held back data, all the while **ensuring customer privacy**_". 
+
+Their [first partnership](https://socialscience.one/our-facebook-partnership) is with Facebook.
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:37]: I find it odd that Facebook major role is only introduced in a small sentence at the end of the paragraph.

File: _pages/encyclopedia/6-research/3-tools.md [hunk 8]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
 
 ## Facebook Social Science One
 
-<!-- à rédiger -->
+[Social Science One](https://socialscience.one/)(SS1) is a non-profit limited liability company (LLC) based on the [work](http://garyking.org/partnerships) of academics Gary King (Harvard Professor and Director of the Harvard Institute for Quantitative Social Science) and Nathaniel Persily (Stanford Law School Professor).
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:23]:
```suggestion
[Social Science One](https://socialscience.one/)(SS1) is a non-profit limited liability company (LLC) founded by Facebook and the [Social Science Research Council](https://www.ssrc.org/)(SSRC), an American non-profit organization. Social Science One is based on the [work](http://garyking.org/partnerships) of academics Gary King (Harvard Professor and Director of the Harvard Institute for Quantitative Social Science) and Nathaniel Persily (Stanford Law School Professor).
```

File: _pages/encyclopedia/6-research/3-tools.md [hunk 9]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
+
+It implements **partnerships between academic researchers and private industry actors** in a unique fashion, granting the former access to relevant information amassed by the latter, who can "_enlist the scientific community to improve their business and produce social good, while protecting their competitive positions_".
+
+The structure of the partnerships enables a **[commission](https://socialscience.one/faq/whats-commission) of trusted academics** to obtain "_relevant company information and systems_" and to **invite other independent scholars** to do "_research in specific areas, following standard peer review protocols, funded by nonprofit foundations, and with no required pre-publication approval by the company_". In addition to access to datasets, academics can benefit from funding.
+
+[Companies](https://socialscience.one/faq/how-can-my-company-work-social-science-one) collecting informative social data can apply by directly [contacting](https://gking.harvard.edu/pages/contact) the co-founders.
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:29]:
```suggestion
```
I don't see the point of this sentence in this summary.

File: _pages/encyclopedia/6-research/3-tools.md [hunk 10]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
 
 ## Facebook Social Science One
 
-<!-- à rédiger -->
+[Social Science One](https://socialscience.one/)(SS1) is a non-profit limited liability company (LLC) based on the [work](http://garyking.org/partnerships) of academics Gary King (Harvard Professor and Director of the Harvard Institute for Quantitative Social Science) and Nathaniel Persily (Stanford Law School Professor).
+
+It implements **partnerships between academic researchers and private industry actors** in a unique fashion, granting the former access to relevant information amassed by the latter, who can "_enlist the scientific community to improve their business and produce social good, while protecting their competitive positions_".
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:25]:
```suggestion
It implements **partnerships between academic researchers and private industry actors** in a unique fashion, granting the former access to relevant information amassed by the latter, who can "_enlist the scientific community to improve their business and produce social good, while protecting their competitive positions_". Currently, **Facebook is the main provider** of information and the only openly publicized [partnership](https://socialscience.one/our-facebook-partnership). 
```

File: _pages/encyclopedia/6-research/3-tools.md [hunk 11]
```diff
@@ -20,7 +20,21 @@ The Mozilla Misinformation Initiative maintains an [open database of research ar
...
+
+[Companies](https://socialscience.one/faq/how-can-my-company-work-social-science-one) collecting informative social data can apply by directly [contacting](https://gking.harvard.edu/pages/contact) the co-founders.
+
+These independent scholars must go through and pass a rigorous peer review process before being allowed to access data and [funding](https://socialscience.one/grant-process). Academics can **apply** for grants by answering formal Requests for Proposals (RFPs) issued by the [Social Science Research Council](https://www.ssrc.org/)(SSRC) and whose criteria are defined by SS1. Funding originates from several charitable foundations but is for now independently administered by the SSRC. The SSRC also leads the peer review process and manages the research along with SS1.
+
+Updates are regularly posted on SS1's [blog](https://socialscience.one/blog).
+
+The process aims to "_understand and ameliorate society’s greatest challenges_" through social research based on useful yet too often held back data, all the while _**ensuring customer privacy**_". 
+
+Their [first partnership](https://socialscience.one/our-facebook-partnership) is with Facebook.
```
Comment (AdrienFines) [inline _pages/encyclopedia/6-research/3-tools.md:37]:
```suggestion
```

**Pull request merged**